assets/tools.json

```diff
@@ -0,0 +1,3671 @@
+        "features": [],
+        "platforms": [
+            "Linux",
+            "Windows"
+        ],
+        "fmiVersions": [
+            "2.0"
+        ],
+        "fmuExport": [
+            "CS"
+        ],
+        "fmuImport": [
+            "CS"
+        ]
+    },
+    {
+        "name": "DAE Tools",
+        "license": "commercial",
+        "url": "https://www.daetools.com",
+        "vendor": "DAE Tools",
+        "vendorURL": "https://www.daetools.com",
+        "description": "",
+        "features": [],
+        "platforms": [
+            "Windows"
+        ],
+        "fmiVersions": [
+            "2.0"
+        ],
+        "fmuExport": [
+            "CS"
+        ],
+        "fmuImport": []
+    },
+    {
+        "name": "Motion",
+        "license": "commercial",
+        "url": "https://www.ansys.com/ko-kr/products/structures/ansys-motion",
+        "vendor": "Ansys",
+        "vendorURL": "https://www.ansys.com/",
+        "description": "Intergrated structure and motion analysis software from VirtualMotion",
+        "features": [],
+        "platforms": [
+            "Windows"
+        ],
+        "fmiVersions": [
+            "1.0"
+        ],
+        "fmuExport": [
+            "CS"
+        ],
+        "fmuImport": []
+    },
+    {
+        "name": "PowerFactory",
+        "license": "commercial",
+        "url": "https://www.digsilent.de/en/",
+        "vendor": "DIgSILENT",
+        "vendorURL": "https://www.digsilent.de/en/",
+        "description": "PowerFactory is a power system analysis software application used for analysing generation, transmission, distribution and industrial systems",
+        "features": [],
+        "platforms": [
+            "Windows"
+        ],
+        "fmiVersions": [
+            "2.0"
+        ],
+        "fmuExport": [
+            "CS",
+            "ME"
+        ],
+        "fmuImport": [
+            "CS",
+            "ME"
+        ]
+    },
+    {
+        "name": "FMI Kit for Simulink",
+        "license": "osi",
+        "url": "https://github.com/CATIA-Systems/FMIKit-Simulink",
+        "logo": "Simulink.png",
+        "vendor": "Dassault Systèmes",
+        "vendorURL": "https://3ds.com/",
+        "description": "FMI Kit for Simulink is a toolbox for MATLAB/Simulink provides an S-function block to import FMUs and two Simulink Coder targets to export Co-Simulation and Model Exchange FMUs.",
+        "features": [],
+        "platforms": [
+            "Windows",
+            "Linux",
+            "macOS"
+        ],
+        "interfaces": [
+            "GUI",
+            "CLI",
+            "library"
+        ],
+        "fmiVersions": [
+            "1.0",
+            "2.0",
+            "3.0"
+        ],
+        "fmuExport": [
+            "CS",
+            "ME"
+        ],
+        "fmuImport": [
+            "CS",
+            "ME"
+        ]
+    },
+    {
```

Comment:
Would you please update for dspace tools:
{
    "name": "ConfigurationDesk",
    "license": "commercial",
    "url": "https://www.dspace.com/go/configurationdesk",
    "logo": "dSPACE.png",
    "vendor": "dSPACE",
    "vendorURL": "https://www.dspace.com/go/fmi",
    "description": "ConfigurationDesk is an intuitive, graphical configuration and implementation tool, ideal for handling applications from small Rapid Control Prototyping (RCP) developments to large Hardware-in-the-Loop (HIL) tests based on dSPACE real-time hardware, such as SCALEXIO or MicroAutoBox III, including the implementation of behavior models and I/O function code.",
    "features": [],
    "platforms": [
      "Linux"
    ],
    "fmiVersions": [
      "2.0"
    ],
    "fmuExport": [],
    "fmuImport": [
      "CS"
    ]
  },
  {
    "name": "MicroAutoBox III",
    "license": "commercial",
    "url": "https://www.dspace.com/go/microautobox3",
    "logo": "dSPACE.png",
    "vendor": "dSPACE",
    "vendorURL": "https://www.dspace.com/go/fmi",
    "description": "MicroAutoBox III is a real-time simulation system for performing fast, in-vehicle function prototyping. Simulations for MicroAutoBox III are implemented with ConfigurationDesk.",
    "features": [],
    "platforms": [
      "Linux"
    ],
    "fmiVersions": [
      "2.0"
    ],
    "fmuExport": [],
    "fmuImport": [
      "CS"
    ]
  },
  {
    "name": "SCALEXIO",
    "license": "commercial",
    "url": "https://www.dspace.com/go/scalexio",
    "logo": "dSPACE.png",
    "vendor": "dSPACE",
    "vendorURL": "https://www.dspace.com/go/fmi",
    "description": "SCALEXIO is a Hardware-in-the-Loop (HIL) integration and simulation platform from dSPACE. Simulations for SCALEXIO are implemented with ConfigurationDesk.",
    "features": [],
    "platforms": [
      "Linux"
    ],
    "fmiVersions": [
      "2.0"
    ],
    "fmuExport": [],
    "fmuImport": [
      "CS"
    ]
  },
  {
    "name": "SYNECT",
    "license": "commercial",
    "url": "https://www.dspace.com/go/synect",
    "logo": "dSPACE.png",
    "vendor": "dSPACE",
    "vendorURL": "https://www.dspace.com/go/fmi",
    "description": "SYNECT is a data management tool from dSPACE that enables you to manage FMUs and Simulink models as well as their dependencies, versions and variants throughout the entire software development process.",
    "features": [],
    "platforms": [
      "Linux",
      "Windows"
    ],
    "fmiVersions": [
      "2.0"
    ],
    "fmuExport": [],
    "fmuImport": [
      "CS"
    ]
  },
  {
    "name": "SystemDesk",
    "license": "commercial",
    "url": "https://www.dspace.com/go/systemdesk",
    "logo": "dSPACE.png",
    "vendor": "dSPACE",
    "vendorURL": "https://www.dspace.com/go/fmi",
    "description": "SystemDesk is an environment for creating virtual ECUs for SIL simulation. Virtual ECUs can be on application level or they can contain up to the complete hardware-independent ECU software.",
    "features": [],
    "platforms": [
      "Windows"
    ],
    "fmiVersions": [
      "2.0"
    ],
    "fmuExport": [
      "CS"
    ],
    "fmuImport": []
  },
  {
    "name": "TargetLink",
    "license": "commercial",
    "url": "https://www.dspace.com/go/targetlink",
    "logo": "dSPACE.png",
    "vendor": "dSPACE",
    "vendorURL": "https://www.dspace.com/go/fmi",
    "description": "TargetLink is a production code generator from dSPACE that generates highly efficient C-code directly from Simulink/Stateflow models.",
    "features": [],
    "platforms": [
      "Linux",
      "Windows"
    ],
    "fmiVersions": [
      "2.0"
    ],
    "fmuExport": [
      "CS"
    ],
    "fmuImport": []
  },
    {
      "name": "VEOS",
      "license": "commercial",
      "url": "https://www.dspace.com/go/fmi",
      "logo": "dSPACE.png",
      "vendor": "dSPACE",
      "vendorURL": "https://www.dspace.com/go/fmi",
      "description": "VEOS is a PC-based virtual integration and simulation platform from dSPACE.",
      "features": [],
      "platforms": [
        "Linux",
        "Windows"
      ],
      "fmiVersions": [
        "1.0",
        "2.0"
      ],
      "fmuExport": [],
      "fmuImport": [
        "CS"
      ]
    },